src/main/java/org/jenkinsci/plugins/DependencyTrack/ApiClientException.java

```diff
@@ -15,7 +15,9 @@
  */
 package org.jenkinsci.plugins.DependencyTrack;
 
-public class ApiClientException extends Exception {
+import java.io.IOException;
+
+public class ApiClientException extends IOException {
```

Comment:
The BuildStep.perform() method expects IOExceptions to be thrown, and as ApiClientExceptions seem like IOExceptions, marking it as such reduces the need for special handling.